src/Illuminate/Foundation/Console/TinkerCommand.php

```diff
@@ -1,11 +1,25 @@
 <?php namespace Illuminate\Foundation\Console;
 
 use Psy\Shell;
+use Psy\Configuration;
 use Illuminate\Console\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Illuminate\Foundation\Console\Tinker\Presenters\EloquentModelPresenter;
+use Illuminate\Foundation\Console\Tinker\Presenters\EloquentCollectionPresenter;
+use Illuminate\Foundation\Console\Tinker\Presenters\IlluminateApplicationPresenter;
 
 class TinkerCommand extends Command {
 
 	/**
+	 * artisan commands to include in the tinker shell.
+	 *
+	 * @var array
+	 */
+	protected static $commandWhitelist = [
+		'clear-compiled', 'down', 'env', 'inspire', 'migrate', 'optimize', 'up',
```

Comment:
Hmmm. Inspire doesn't make sense. Users are likely to have deleted this command from their application.